### ISSUE TITLE
Check & Checklist: Double sets of inputs read out

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#793]
### Expected behavior

Checkboxes and radio buttons have one input element which has a label.
### Actual behavior

Screenreaders have access to both input and span with check/radio elements. Only one is clickable in the DOM with mouse/keyboard but both are read out by screenreader in the browser Accessibility API.  A screenreader hears two sets of inputs which can be confusing and annoying to hear repeats. The span element doesn't have an associated label make it more unclear what this second input is and which to use.  
### Steps to reproduce

Turn on Voiceover and move through radiobuttons and checkboxes

**Comment by Blackbaud-PatrickOFriel** [comment]
Yeah I think we can just remove the role="checkbox" on the span, I don't think it's necessary.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.3](https://github.com/blackbaud/skyux/releases/tag/v1.7.3)
